sale_project_fixed_price_task_completed_invoicing/models/sale_order_line.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    @api.model
+    def create(self, vals):
```

Comment:
@angelmoya It takes the logic from `sale_timesheet` and overrides it in order to make it work with the new selection in `line.product_id.track_service` You'll find there the commit from odoo implementing it.  https://github.com/odoo/odoo/commit/90130744e04b4a15973a484a78d461208920e510